### PR TITLE
Add UserAgent or: Make tests work

### PR DESCRIPTION
**Pull request by stefan-balke**

### CODE REVIEW

**Comment by stefan-balke** [comment]
Okay, tests pass now.

`archive.yaml` is up for discussion though.

File: tests/test_datasets.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 import pytest
+from fake_useragent import UserAgent
 
 
-d = yaml.load(open("mir-datasets.yaml"))
+# get fake user agent
+ua = UserAgent()
+
+# open dataset file
+with open('mir-datasets.yaml', 'rb') as hdl:
```
Comment (ejhumphrey) [inline tests/test_datasets.py]: I would expect this to be `'r'`, rather than `'rb'`, but I'm guessing this works?

**Comment by ejhumphrey** [comment]
I'm not totally clear on the purpose of `archive.yaml`, so yea, maybe worth discussing? It doesn't seem to be used in this PR? If nothing else, I'd propose adding information to the main readme to describe its intended purpose... but otherwise, this all looks fine to me.

**Comment by ejhumphrey** [comment]
green is clean :shipit:

**Pull request merged**